code/extensions/GoogleSitemapExtension.php

```diff
@@ -18,6 +18,15 @@ public function canIncludeInGoogleSitemap()
 
         if ($this->owner->hasMethod('AbsoluteLink')) {
             $hostHttp = parse_url(Director::protocolAndHost(), PHP_URL_HOST);
+
+            // Subsite support
+            if (class_exists('Subsite')) {
```

Comment:
Subsites has namespaces now - probably worth adding those to this PR too

Reply:
I've changed the base branch to 1.6, so at this point the Subsites support is for 3.x only.